katello/service-wait

```diff
@@ -57,7 +57,7 @@ before_start() {
 # after start or restart
 after_start() {
   case "$SERVICE" in
-    tomcat6|tomcat7)
+    tomcat|tomcat6|tomcat7)
```

Comment:
A Redmine issue to pull this in would be nice as well.
On May 20, 2016 6:15 PM, "Stephen Benjamin" <notifications@github.com>
wrote:

> In katello/service-wait
> <https://github.com/Katello/katello-packaging/pull/229#discussion_r64114823>
> :
>
> > @@ -57,7 +57,7 @@ before_start() {
> >  # after start or restart
> >  after_start() {
> >    case "$SERVICE" in
> > -    tomcat6|tomcat7)
> > +    tomcat|tomcat6|tomcat7)
>
> I think tomcat7 can actually be removed, not sure that's the name of any
> service
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Katello/katello-packaging/pull/229/files/76bb064d9dcef0b9e488b184b7ca4b796a563c41#r64114823>
>